src/query/functions/tests/it/aggregates/testdata/agg.txt

```diff
@@ -823,18 +823,18 @@ evaluation (internal):
 | Column | Data                                                                    |
 +--------+-------------------------------------------------------------------------+
 | x_null | NullableColumn { column: UInt64([1, 2, 3, 4]), validity: [0b____0011] } |
-| Output | NullableColumn { column: Float64([1]), validity: [0b_______1] }         |
+| Output | NullableColumn { column: Float64([1.8]), validity: [0b_______1] }       |
 +--------+-------------------------------------------------------------------------+
 
 
 ast: quantile_cont(0.8)(a)
 evaluation (internal):
-+--------+-----------------------------------------------------------------+
-| Column | Data                                                            |
-+--------+-----------------------------------------------------------------+
-| a      | Int64([4, 3, 2, 1])                                             |
-| Output | NullableColumn { column: Float64([1]), validity: [0b_______1] } |
-+--------+-----------------------------------------------------------------+
++--------+-------------------------------------------------------------------+
+| Column | Data                                                              |
++--------+-------------------------------------------------------------------+
+| a      | Int64([4, 3, 2, 1])                                               |
+| Output | NullableColumn { column: Float64([3.4]), validity: [0b_______1] } |
++--------+-------------------------------------------------------------------+
```

Comment:
cc @ariesdevil  so the `quantile_cont` is wrong without this pr ?

Reply:
`quantile_cont` compute results are correct in actual execution, but it is wrong in agg.txt.